packages/now-next/test/fixtures/22-ssg-v2-shared-lambdas/now.json

```diff
@@ -0,0 +1,226 @@
+{
+  "version": 2,
+  "builds": [
+    {
+      "src": "package.json",
+      "use": "@vercel/next",
+      "config": {
+        "sharedLambdas": true
```

Comment:
How will a user know if/when to assign this? Or is the intention to be a feature flag until its rolled out to everyone?

Reply:
Yeah it's meant to be a feature flag for now while we test it out